src/webextension/list/manage/components/account-summary-label.css

```diff
@@ -3,10 +3,24 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 .account-summary {
+  display: flex;
+  flex-flow: row nowrap;
+  align-items: center;
+  align-self: center;
```

Comment:
Let's do this in the CSS for `<Toolbar/>` (`src/webextension/widgets/toolbar.css`); currently, it has `align-items: baseline`, but `align-items: center` would probably be better...